server/utils/placementRequests/utils.ts

```diff
@@ -32,3 +35,9 @@ export const searchButton = (placementRequest: PlacementRequest) =>
     { id: placementRequest.id },
     { text: 'Search', attributes: { class: 'govuk-button' } },
   )
+
+export const applicationLink = (placementRequest: PlacementRequest, text: string, hiddenText: string) =>
+  linkTo(applyPaths.applications.show, { id: placementRequest.applicationId }, { text, hiddenText })
```

Comment:
The type is generated a bit weirdly. It seems that it's got all the attributes as optional, but then writes over them with a type where all of them are required, so `applicationId` etc will always exist. This reflects what's in the OpenAPI spec too https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/src/main/resources/static/api.yml#L4665. I think it's because the `required` property should be embedded in the second type. Will open a PR.